### PR TITLE
Upgrade 54 Patches

### PR DESCRIPTION
**Pull request by rachel-bousfield**
Fixes a chain params bug and improves style

### CODE REVIEW

**Review by hkalodner — APPROVED** [review]
LGTM

File: arb_os/gasAccounting.mini [hunk 1]
```diff
@@ -228,18 +228,18 @@ view throw func initCongestionState(isInit: bool) -> CongestionState {
                     gasPoolMax * x / 100
                 }
             } else {
-                gasPoolMax / 2
+                const::Default_Param_GasPoolFullnessTargetPercent
```
Comment (edfelten) [inline arb_os/gasAccounting.mini]: This needs to be multiplied by `gasPoolMax` then divided by 100.

**Review by edfelten — CHANGES_REQUESTED** [review]
One bug to fix.

File: arb_os/gasAccounting.mini (hunk 1 above)
Comment (rachel-bousfield) [reply arb_os/gasAccounting.mini]: Nice catch, fixed

**Review by edfelten — APPROVED** [review]
Looks good.

**Pull request merged**